WordPressSecurityWhitePaper.html

```diff
@@ -292,6 +550,25 @@ <h3>Footnotes</h3>
 	<li id='footnote30'><a href="#ref30">[30]</a> <a href="https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards">https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards</a></li>
+	<li id='footnote31'><a href="#ref31">[31]</a> <a href="https://developer.wordpress.org/reference/functions/esc_html/">https://developer.wordpress.org/reference/functions/esc_html/</a></li>
+	<li id='footnote32'><a href="#ref32">[32]</a> <a href="https://developer.wordpress.org/reference/functions/esc_attr/">https://developer.wordpress.org/reference/functions/esc_attr/</a></li>
+	<li id='footnote33'><a href="#ref33">[33]</a> <a href="https://developer.wordpress.org/reference/functions/esc_url/">https://developer.wordpress.org/reference/functions/esc_url/</a></li>
+	<li id='footnote34'><a href="#ref34">[34]</a> <a href="https://developer.wordpress.org/reference/functions/esc_js/">https://developer.wordpress.org/reference/functions/esc_js/</a></li>
+	<li id='footnote35'><a href="#ref35">[35]</a> <a href="https://developer.wordpress.org/reference/functions/wp_kses/">https://developer.wordpress.org/reference/functions/wp_kses/</a></li>
+	<li id='footnote36'><a href="#ref36">[36]</a> <a href="https://developer.wordpress.org/reference/functions/wp_kses_post/">https://developer.wordpress.org/reference/functions/wp_kses_post/</a></li>
+	<li id='footnote37'><a href="#ref37">[37]</a> <a href="https://developer.wordpress.org/reference/functions/wp_kses_data/">https://developer.wordpress.org/reference/functions/wp_kses_data/</a></li>
+	<li id='footnote38'><a href="#ref38">[38]</a> <a href="https://developer.wordpress.org/reference/classes/wpdb/prepare/">https://developer.wordpress.org/reference/classes/wpdb/prepare/</a></li>
+	<li id='footnote39'><a href="#ref39">[39]</a> <a href="https://developer.wordpress.org/reference/classes/wpdb/update/">https://developer.wordpress.org/reference/classes/wpdb/update/</a></li>
+	<li id='footnote40'><a href="#ref40">[40]</a> <a href="https://developer.wordpress.org/reference/classes/wpdb/insert/">https://developer.wordpress.org/reference/classes/wpdb/insert/</a></li>
+	<li id='footnote41'><a href="#ref41">[41]</a> <a href="https://developer.wordpress.org/reference/functions/sanitize_email/">https://developer.wordpress.org/reference/functions/sanitize_email/</a></li>
+	<li id='footnote42'><a href="#ref42">[42]</a> <a href="https://developer.wordpress.org/reference/functions/sanitize_html_class/">https://developer.wordpress.org/reference/functions/sanitize_html_class/</a></li>
+	<li id='footnote43'><a href="#ref43">[43]</a> <a href="https://wordpress.org/about/stats/">https://wordpress.org/about/stats/</a></li>
+	<li id='footnote44'><a href="#ref44">[44]</a> <a href="https://wordpress.org/about/privacy/">https://wordpress.org/about/privacy/</a></li>
+	<li id='footnote45'><a href="#ref45">[45]</a> <a href="http://wordpressfoundation.org/trademark-policy/">http://wordpressfoundation.org/trademark-policy/</a></li>
```

Comment:
```suggestion
	<li id='footnote45'><a href="#ref45">[45]</a> <a href="https://wordpressfoundation.org/trademark-policy/">https://wordpressfoundation.org/trademark-policy/</a></li>
```

can use https instead of http link.